inference-services/kfserving/template.sh

```diff
@@ -15,13 +16,15 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: "${ORG}-${PROJECT}-kfserving"
+  namespace: fuseml-workloads
 secrets:
   - name: "${ORG}-${PROJECT}-storage"
 ---
 apiVersion: "serving.kubeflow.org/v1beta1"
 kind: "InferenceService"
 metadata:
   name: "${ORG}-${PROJECT}"
+  namespace: fuseml-workloads
```

Comment:
do you really need to hardcode the namespace here? This container is executed by the service account defined at the triggers template, so I expect it to by default set the context to `fuseml-workloads`.

Reply:
I hardcoded it here because it's a required for application. I believe I will need it when deleting the resource (https://github.com/fuseml/fuseml-core/pull/31/files#diff-6aa5d1a73434cb01261342c0d7bb9fc6a6e22da341bb5874489d60119de42ce2R42)

I was thinking about other options:

- having it passed as argument from tekton, and replacing it in the template
  (BTW I will also need the _worklow name_ as argument anyway, where do you think I should put it, as Tekton input argument, or save it to the file just like you do with ORG and PROJECT)
- let application fetch the namespace from workflow definition, is it possible now?

What do you think?

Reply:
So, I've changed the way the namespace is read, but it does not work.

What do you think, could I just add it as env variable to the tekton task when generating pipeline? (To make the code simpler, it could be added to _all_ tasks)... Same with that workflow name...

Reply:
yes, I am fine with that